<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesTransform</class>
 <widget class="QWidget" name="PropertiesTransform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>160</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxRotationZ">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>63</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-360.000000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelSx">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>96</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="../gameplay-editor.qrc">:/res/images/axis-x.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelPosition">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>36</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Position</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPz">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>36</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="../gameplay-editor.qrc">:/res/images/axis-z.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelRotation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>66</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotation</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxRotationY">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>63</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-360.000000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelScale">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>96</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRx">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>66</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="../gameplay-editor.qrc">:/res/images/axis-x.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelPx">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>36</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="../gameplay-editor.qrc">:/res/images/axis-x.png</pixmap>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxPositionX">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>33</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxPositionZ">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>33</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxRotationX">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>63</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-360.000000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxScaleZ">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>93</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxScaleX">
   <property name="geometry">
    <rect>
     <x>67</x>
     <y>93</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxScaleY">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>93</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelPy">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>36</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="../gameplay-editor.qrc">:/res/images/axis-y.png</pixmap>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxPositionY">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>33</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelRy">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>66</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="../gameplay-editor.qrc">:/res/images/axis-y.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelSy">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>96</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="../gameplay-editor.qrc">:/res/images/axis-y.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelRz">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>66</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="../gameplay-editor.qrc">:/res/images/axis-z.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelSz">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>96</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="../gameplay-editor.qrc">:/res/images/axis-z.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStatic">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>125</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string> Static</string>
   </property>
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/static.png</normaloff>:/res/images/static.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMovable">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>125</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>  Movable</string>
   </property>
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/movable.png</normaloff>:/res/images/movable.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelMobility">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Mobility</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeaderIcon">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>5</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../gameplay-editor.qrc">:/res/images/scene-object.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHeader">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeaderName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>5</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Transform</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonHeaderOptions">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>4</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/menu.png</normaloff>:/res/images/menu.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonHeaderExpander">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>2</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/arrow-down.png</normaloff>:/res/images/arrow-down.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
  </widget>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/reset.png</normaloff>:/res/images/reset.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/help.png</normaloff>:/res/images/help.png</iconset>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <zorder>pushButtonHeader</zorder>
  <zorder>doubleSpinBoxRotationZ</zorder>
  <zorder>labelSx</zorder>
  <zorder>labelPosition</zorder>
  <zorder>labelPz</zorder>
  <zorder>labelRotation</zorder>
  <zorder>doubleSpinBoxRotationY</zorder>
  <zorder>labelScale</zorder>
  <zorder>labelRx</zorder>
  <zorder>labelPx</zorder>
  <zorder>doubleSpinBoxPositionX</zorder>
  <zorder>doubleSpinBoxPositionZ</zorder>
  <zorder>doubleSpinBoxRotationX</zorder>
  <zorder>doubleSpinBoxScaleZ</zorder>
  <zorder>doubleSpinBoxScaleX</zorder>
  <zorder>doubleSpinBoxScaleY</zorder>
  <zorder>labelPy</zorder>
  <zorder>doubleSpinBoxPositionY</zorder>
  <zorder>labelRy</zorder>
  <zorder>labelSy</zorder>
  <zorder>labelRz</zorder>
  <zorder>labelSz</zorder>
  <zorder>pushButtonStatic</zorder>
  <zorder>pushButtonMovable</zorder>
  <zorder>labelMobility</zorder>
  <zorder>labelHeaderIcon</zorder>
  <zorder>labelHeaderName</zorder>
  <zorder>toolButtonHeaderOptions</zorder>
  <zorder>toolButtonHeaderExpander</zorder>
 </widget>
 <resources>
  <include location="../gameplay-editor.qrc"/>
 </resources>
 <connections/>
</ui>
